### DISCUSSION TITLE
设置选中节点/边的附加显示内容 不生效

### DISCUSSION THREAD

**Discussion by avrinfly** [antvis/X6#1831] [Q&A]
![image](https://user-images.githubusercontent.com/30068005/154797780-c6866574-f9ea-4625-9e9e-8fa28c728045.png)
根据上面文档里的写法，我试了两个写法，发现在画布里不生效，有大佬知道是什么问题吗？能给写一个示例就更好了

![image](https://user-images.githubusercontent.com/30068005/154798032-d13d86fd-77d6-4aea-b181-163941b3fde9.png)

![image](https://user-images.githubusercontent.com/30068005/154798047-664bca00-2ee6-41e8-91f7-a0fd77669df6.png)

期望是实现下面的效果，有大佬有思路吗
![image](https://user-images.githubusercontent.com/30068005/154798272-c8cecf0a-9eb5-4da0-946d-3f88053c88c7.png)

**Comment by NewByVector** [comment]
直接 `graph.setSelectionDisplayContent('xxx')` 即可。
